src/app/ApplicationTemplate.Access/ApplicationTemplate.Access.csproj

```diff
@@ -24,9 +24,13 @@
 		<PackageReference Include="Refit" Version="7.0.0" />
 		<PackageReference Include="ReviewService.Abstractions" Version="1.0.0" />
 		<PackageReference Include="System.Text.Json" Version="7.0.3" />
-		<PackageReference Include="MallardMessageHandlers" Version="0.2.0-dev.21" />
+		<PackageReference Include="MallardMessageHandlers" Version="1.2.0" />
 		<PackageReference Include="Nventive.Persistence.Reactive" Version="0.5.0" />
 		<PackageReference Include="Nventive.Persistence" Version="0.5.0" />
 		<PackageReference Include="System.Reactive" Version="5.0.0" />
 	</ItemGroup>
+
+	<ItemGroup>
+	  <PackageReference Update="GooseAnalyzers" Version="1.0.0" />
```

Comment:
Don't add a second reference to GooseAnalyzers. (It's already referenced in Directory.Build.props)